src/main/java/by/exadel/internship/service/InternshipService.java

```diff
@@ -0,0 +1,41 @@
+package by.exadel.internship.service;
+
+
+import by.exadel.internship.dto.internshipDTO.GuestInternshipDTO;
+import by.exadel.internship.entity.Internship;
+import by.exadel.internship.mapper.InternShipMapper;
+import by.exadel.internship.repository.InternshipRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.UUID;
+
+@Service
+public class InternshipService {
+
+    private final InternshipRepository internshipRepository;
+    private final InternShipMapper internShipMapper;
+
+    @Autowired
+    public InternshipService(InternshipRepository internshipRepository, InternShipMapper internShipMapper) {
+        this.internshipRepository = internshipRepository;
+        this.internShipMapper = internShipMapper;
+    }
+
+    public GuestInternshipDTO getById(UUID uuid) {
+
+        Internship internship = internshipRepository.getOne(uuid);
```

Comment:
Will be better to use this method in case internship is not found and throw exception
```suggestion
        Internship internship = internshipRepository.findById(uuid);
```

Reply:
![image](https://user-images.githubusercontent.com/65280608/113557934-78c44c00-9607-11eb-90d5-974b7268f8cc.png)

Reply:
done
![image](https://user-images.githubusercontent.com/65280608/113572671-593a1d00-9621-11eb-83f0-ea53b19f2490.png)
![image](https://user-images.githubusercontent.com/65280608/113572742-6e16b080-9621-11eb-984a-8173d2a979bd.png)

Reply:
@Maksim-maevsky please, provide your own exception with annotation  @ResponseStatus (404) and throw it via method `orElseThrow`
https://www.baeldung.com/exception-handling-for-rest-with-spring#3-responsestatusexceptionresolver

Reply:
![image](https://user-images.githubusercontent.com/65280608/113664773-06aa4080-96b5-11eb-9fde-5ac86f1f2f5e.png)